### DISCUSSION TITLE
Make the Argon and Triangles skins downloadable or available for Stable

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#23182] [Ideas] [answered]
Hello,

The default skin for osu!(stable) looks very old and it's hard to find a skin as smooth and modern as Argon (This is just a personal opinion).
Some people don't like the default skin for Stable and it would be a nice addition to either make Argon and Triangles available for download or add them as built-in skins to Stable.

**Comment by ItsShamed** [comment, marked as answer]
These skins are hard-coded into the game, and since stable is pretty much feature-locked, they won't be added in stable anytime soon (pretty much impossible).

**Reply by Schweeeeeeeeeeeeeeee** [reply]
Making it downloadable as an osk file would be nice for easy customization, though, I guess.

**Reply by ItsShamed** [reply]
As I said the skin is litteraly hard-coded into the game's code, you cannot even make it downloadable.

**Reply by yesseruser** [reply]
I meant re-create it for download, idc about the menus just gameplay elements

**Reply by ItsShamed** [reply]
This is what I found by [doing a little Google search](https://www.google.com/search?q=osu+stable+Argon+skin)

https://www.reddit.com/r/OsuSkins/comments/y11vk2/lazer_argon_skin_remake_std_gameplay_only/

https://osu.ppy.sh/community/forums/topics/1733101?n=1

Took me less than a minute.

**Reply by yesseruser** [reply]
Okay, thank you.
